models/src_zendesk.yml

```diff
@@ -102,14 +102,13 @@ sources:
             description: Index number of the domain name associated with the organization
 
       - name: group
-        identifier: "{%- if target.type == 'snowflake' -%} {{ var('zendesk_group_identifier', 'GROUP') }} {%- else -%} {{ var('zendesk_group_identifier', 'group') }} {%- endif -%}"
+        identifier: >
+          {% if target.type == 'snowflake' %}{{ var('zendesk_group_identifier', '"GROUP"') }}{% else %}{{ var('zendesk_group_identifier', 'group') }}{% endif %}
```

Comment:
With this adjustment, we should also make the corresponding change in the README.

See how we do this in Salesforce Source for [reference](https://github.com/fivetran/dbt_salesforce_source#-snowflake-users).